### ISSUE TITLE
Update variation normalizer version to support N DNA sequence symbols

### ISSUE THREAD

**Issue by theferrit32** [clingen-data-model/architecture#546]
https://github.com/cancervariants/variation-normalization/commit/459b81942ab320191bbd2c5aa409d9d27c2be9b7

**Comment by theferrit32** [comment]
Also need to update to https://github.com/cancervariants/variation-normalization/releases/tag/0.7.0-dev7 to incorporate something changed in MANE's representation of `GRCh38_chr`. If `0.7.0-dev7` might include the above commit for N symbols.

**Comment by theferrit32** [comment]
Resolved by #559

**Comment by theferrit32** [comment]
Verified with 
input:
`http://localhost:8002/variation/translate_from?variation=NC_000001.11%3A193135607%3A%3ATTTTTTTTNNNNNNN&fmt=spdi`
output:
```
{
  "query": {
    "variation": "NC_000001.11:193135607::TTTTTTTTNNNNNNN",
    "fmt": "spdi"
  },
  "warnings": [],
  "service_meta_": {
    "version": "0.7.0-dev7",
    "response_datetime": "2023-05-05T18:05:34.231058",
    "name": "variation-normalizer",
    "url": "https://github.com/cancervariants/variation-normalization"
  },
  "vrs_python_meta_": {
    "version": "0.9.0a0",
    "name": "vrs-python",
    "url": "https://github.com/ga4gh/vrs-python"
  },
  "variation": {
    "id": "ga4gh:VA.ZEwjzMRt8PkwkQk5H4y-XzOZMMeQs0QI",
    "type": "Allele",
    "location": {
      "id": null,
      "type": "SequenceLocation",
      "sequence_id": "ga4gh:SQ.Ya6Rs7DHhDeg7YaOSg1EoNi3U_nQ9SvO",
      "start": {
        "type": "Number",
        "value": 193135607
      },
      "end": {
        "type": "Number",
        "value": 193135607
      }
    },
    "state": {
      "type": "LiteralSequenceExpression",
      "sequence": "TTTTTTTTNNNNNNN"
    }
  }
}
```
